connector_search_engine/models/se_binding.py

```diff
@@ -166,7 +166,7 @@ def recompute_json(self, force_export: bool = False):
                 with self.env.cr.savepoint():
                     record.data = index.model_serializer.serialize(record.record)
                     record.date_recomputed = fields.Datetime.now()
-            except Exception as e:
+            except UserError as e:
```

Comment:
@sbejaoui In this case you'll only accept Odoo's exceptions but not for exemple pydantic exceptions. IMO, we must catch any exception except the one from pg....
```suggestion
from psycopg2.errors import Error

...
            try:
                ....

            except Error as pg_error:
                # PG error could make the cursor unusable
                raise pg_error

            except Exception as e:
                record.state = "recompute_error"
                record.error = str(e)
                continue
```